app/src/dkg.c

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
```

Comment:
later we can either move this file to a handler/ folder where all apdu handlers live, or add a prefix/suffix with the handler keyboard, but this is an enhancement for later

Reply:
Great!